src/controller/shop.controller.ts

```diff
@@ -71,6 +71,14 @@ export default class ShopController extends BaseController {
     this.success(res, '--shop/deleted', 'shop deleted', 200, null);
   }
 
+  async getAllShops(req: Request, res: Response) {
+    const shops = await prisma.shop.findMany();
```

Comment:
this is wrong.. you can't return all shops available on the app.. you only need to return shops created by the user.